src/App.tsx

```diff
@@ -1,26 +1,105 @@
-import React from 'react';
-import logo from './logo.svg';
-import './App.css';
+import { FC, ReactNode, useEffect, useState } from 'react';
+
+import { useGetReposByLoginQuery, useGetUserByLoginQuery } from 'api/user.api';
+import DefaultWindow from 'components/layout/DefaultWindow';
+import Header from 'components/layout/Header';
+import Profile from 'components/layout/Profile';
+import Repositories from 'components/layout/Repositories';
+import Container from 'components/shared/Container';
+import 'styles/components/app.scss';
+
+const App: FC = () => {
+  const [searchProfile, setSearchProfile] = useState('');
+  const [currentPage, setCurrentPage] = useState(1);
+  const [isLoading, setIsLoading] = useState(false);
```

Comment:
I thought that only in this case there will be a re-render and preloader, but I was wrong. since we get data already from the hook, only from RTK query and dynamically pass props.